qa-covid19.planx-pla.net/portal/gitops.json

```diff
@@ -378,85 +378,107 @@
     },
     {
       "tabTitle": "Summary",
-      "charts": {},
-      "filters": {
-        "tabs": [{
-            "title": "Reports",
-            "fields": [
-              "staffed_all_beds",
-              "staffed_icu_beds",
-              "licensed_all_beds",
-              "all_bed_occupancy_rate",
-              "icu_bed_occupancy_rate",
-              "population",
-              "population_gtr_20",
-              "population_gtr_65",
-              "estimated_full_mech_ventilators",
-              "estimated_full_mech_pediatric_ventilators"
-            ]
-          },
-          {
-            "title": "Location",
-            "fields": [
-              "county",
-              "province_state"
-            ]
-          }
-        ]
-      },
-      "table": {
-        "enabled": false,
-        "fields": []
+      "charts": {
+        "country_region": {
+          "chartType": "pie",
```

Comment:
can we try a bar chart instead of a pie chart? it doesn't look great since it's mostly US

![Screen Shot 2020-06-08 at 1 30 16 PM](https://user-images.githubusercontent.com/4224001/84066907-3b157900-a98c-11ea-8334-3e9f7f408fcb.png)